samples/led/main.c

```diff
@@ -1,22 +1,15 @@
+#include <hal/led.h>
 #include <hal/xbox.h>
-#include <xboxkrnl/xboxkrnl.h>
-
-#define SMC_REG_LEDMODE             0x07
-#define SMC_REG_LEDSEQ              0x08
 
 void main(void)
 {
-    // Set LED to be user controlled
-    HalWriteSMBusValue(0x20, SMC_REG_LEDMODE, FALSE, 0x01);
+  // Set front LEDs to shine green and then red.
+  XSetCustomLED(LED_OFF, LED_OFF, LED_OFF, LED_OFF);
```

Comment:
Why is this OFF, OFF, OFF, OFF; if it's meant to be `green and then red`. Also maybe make clear that this is a repeating pattern.

Reply:
You're right, I forgot I had made a change in order to help @Ernegien troubleshoot. Fixed!